xml/ha_migration.xml

```diff
@@ -708,6 +724,90 @@
    </important>
    <para>The &hawk2; <guimenu>Status</guimenu> screen also shows a warning if
     different CRM versions are detected for your cluster nodes.</para>
+   <para>Beside an in-place upgrade, many customers prefer a fresh installation
+    even for moving to the next service pack. The following procedure shows a
+    scenario where a two-node cluster with the nodes &node1; and &node2; is
+    upgraded to the next service pack (SP):
+   </para>
+   <procedure xml:id="pro-ha-migration-rolling-upgrade-newsp-freshinstall">
+    <title>Performing a Cluster-wide Fresh Installation of a New Service Pack</title>
+    <step xml:id="st-ha-migration-rolling-up-sp-backup">
+     <para>Make a backup of your cluster configuration. A minimum set of files
+      are shown in the following list<!--<xref linkend="ex.ha.migration.minlist4backup"
+       xrefstyle="select:label"/>-->:
+     </para>
+     <!--<example xml:id="ex.ha.migration.minlist4backup">
+      <title>Minimum Files for Backup</title>-->
+     <screen>/etc/corosync/corosync.conf
+/etc/corosync/authkey
+/etc/sysconfig/sbd
+/etc/modules-load.d/watchdog.conf
+/etc/hosts
+/etc/ntp.conf</screen>
+     <!--</example>-->
+     <para>Depending on your resources, you may also need the following files:</para>
+     <!--<example xml:id="ex.ha.migration.extlist4backup">
+      <title>Extended List of Files for Backup</title>-->
+     <screen>/etc/services
+/etc/passwd
+/etc/shadow
+/etc/groups
+/etc/drbd/*
+/etc/lvm/lvm.conf
+/etc/mdadm.conf
+/etc/mdadm.SID.conf</screen>
+     <!--</example>-->
+    </step>
+    <step xml:id="st-ha-migration-rolling-alice">
+     <para>Start with node &node1;.
+     </para>
+     <substeps>
+      <step>
+       <para>
+        Put the node into standby node to move all resources to the
+        other node:
```

Comment:
I absolutely agree with your findings! :+1:

I used the wording from the link you've added:

    Put the node into standby node. That way, services can migrate off the node:

Reply:
I should've been clearer :-) The only problem with this description is the wordings :-)

To be precise, it's the "resources" rather than "services" being managed by the cluster. So that users don't get confused with "cluster services".

Another one is, we use "move" rather than "migrate" for such an occasion. "migrate" has been used in too many occasions which introduces a lot confusions ...

Could we improve this description probably together/later also for the one in other place?

Reply:
Ahh, I'm sorry. I used the phrase from the link which still mentions "migrate off".

    Put the node into standby node. That way, resources can move off the node:


> Could we improve this description probably together/later also for the one in other place?

Yes, I would do that in a separate pull request to keep this clean (as this one needs to be cherry-picked in different maintenance branches).

Reply:
Integrated in commit 2586002.